src/modules/user/view/forgot_password.hbs

```diff
@@ -0,0 +1,24 @@
+<style>
+ .message-content {
+    font-family: Arial, Helvetica, sans-serif;
+    max-width: 600px;
+    font-size: 18px;
+    line-height: 21px;
+ }
+</style>
+
+<div class="message-content">
+  <p>Olá, {{name}}</p>
+  <p>Parece que uma troca de senha para sua conta foi solicitada</p>
+
+  <p>Se foi você, então clique no link abaixo para escolher uma nova senha:</p>
+  <p>
+    <a href="{{link}}">Resetar minha senha</a>
+  </p>
+
```

Comment:
I think it is valid for user accessibility to add:

Está com problemas para acessar o link acima? Use este link:
{{link}} 

Para sua segurança, este link irá expirar em x horas ou imediatamente após você redefinir sua senha.

Reply:
done !